.github/workflows/ci.yaml

```diff
@@ -0,0 +1,27 @@
+name: CI
+on:
+  push:
+  pull_request:
+    types: [opened, reopened]
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
+jobs:
+  Security:
+    name: Security Pipeline
+    uses: uc-cdis/.github/.github/workflows/securitypipeline.yaml@master
+    with:
+       python-poetry: 'false'
+    secrets: inherit
+
+  UnitTest:
+    name: Python Unit Test with Postgres
+    uses: uc-cdis/.github/.github/workflows/python_unit_test.yaml@master
+    with:
+       python-version: '3.9'
+       test-script: 'tests/ci_commands_script.sh'
+  Coveralls:
```

Comment:
i don't think this will work because it's set up as a different "job", when it should be a new "step" in the same "job" (to use the report file generated by the `pytest` command). Unfortunately i don't think you can add a step to a job that's defined in a reusable workflow, so if you want to do it that way, it would have to be directly in the reusable workflow.

What i was suggesting was to add the `coveralls` command to `tests/ci_commands_script.sh`. But i think we'd need to pass the token to the `test-script`.

I see now that it's not as straightforward as i thought 😅

Reply:
@k-burt-uch 
https://github.com/uc-cdis/.github/pull/68
https://github.com/uc-cdis/indexd/compare/fix/content-dates-for-new-version...fix/content-dates-for-new-version-pauline